app/services/permission_service.rb

```diff
@@ -49,6 +50,7 @@ def allow?(controller, action)
       return true if controller == 'cart'        && action == 'show'
       return true if controller == 'cart_cars'   && action.in?(%w(create destroy update))
       return true if controller == 'orders'      && action == 'create'
+      return true if controller == 'users'       && action == 'profile'
```

Comment:
Operator `&&` should be surrounded with a single space.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.